### PR TITLE
[4.11] Set up release controller configs and their deps

### PR DESCRIPTION
**Pull request by petr-muller**
1. Create release-controller configurations for 4.11. Make copies of 4.10 ones, make them hidden (so they are listed at the bottom of release-controller UI) and make them trigger just one of the configured jobs (no point in running many jobs on 4.11 for now)
2. ~Setting up release-controller configurations makes the tooling to generate openshift mirroring lists~ MOVED TO A SEPARATE FOLLOWUP PR
3. Populate release jobs. All intervals/crons are set to `@weekly/168h` for now: running 4.11 jobs often does not have much value
4. Jobs that need a stable 4.10 are using latest accepted nightly as a stand-in
5. Adding release jobs makes the automation generate manifests for various release-controller bits (imagestreams etc).

Changes are added in individual commits to make maintenance (rebasing...) and reviews easier.

/cc @hongkailiu @openshift/test-platform 
/hold

### CODE REVIEW

File: clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml
Comment (hongkailiu) [inline clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: nit: would be nicer with `project`. (I knew this is controlled by a python script.)

**Review by hongkailiu — APPROVED** [review]

File: clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml
Comment (petr-muller) [reply clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: Yeah, unfortunately these bits are generated - so at least it's not hard to maintain them.

Comment (bradmwilliams) [reply clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: I'm more than happy to take a look at this if it's something that can be improved.  Im just not sure I understand what you mean by `would be nicer with project`?

Comment (bradmwilliams) [reply clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: We had to move to auto-generating these pieces because mistakes, while manually editing these files, were killing us.  Certainly willing to make any enhancements to make the whole process easier!

Comment (petr-muller) [reply clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: I think Hongkai means `projected` volumes which allow you to project multiple sources into a single volume and you can then mount it at once. We did it in https://github.com/openshift/release/pull/24593/commits/ce98548fe4577e3ece00a3cef666fb4ee7e86612

Comment (bradmwilliams) [reply clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: Gotcha!  Thanks for the clarification.

**Comment by petr-muller** [comment]
I realized that I can actually move the image mirroring change to a separate PR that will be a followup of this one, MIrroring changes need Clayton's approval so having that here would be quite a burden for merge, and that is unnecessary. So for now I left that change out.

**Comment by petr-muller** [comment]
/hold
Now the PR is auto-approved because I am a root approver, so I'll hold it so we don't accidentaly merge it after a drive-by LGTM or something. I'd like TRT to be aware and have some say about if they like the setup for 4.11.

**Review by bradmwilliams — APPROVED** [review]

File: clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml
Comment (bradmwilliams) [reply clusters/app.ci/release-controller/deploy-ocp-arm64-controller.yaml:244]: I think this should do what you've suggested:
https://github.com/openshift/release/pull/24665

**Comment by petr-muller** [comment]
/remove-lifecycle stale

I need to revive this, code freeze is getting closer

**Comment by petr-muller** [comment]
Instead of rebasing this PR to the current state, I will replace it with a series of smaller PRs that are easier to adapt and review.

**Pull request closed without merging**
